packages/realm/bindgen/package.json

```diff
@@ -1,8 +1,5 @@
 {
-  "name": "@realm/js-binding",
+  "name": "@realm/js-bindgen",
```

Comment:
This is not the binding, but the JS specific tool / package used to generate it.